src/js/components/agencyV2/visualizations/totalObligationsOverTime/paths/Path.jsx

```diff
@@ -50,7 +52,12 @@ const Path = ({
         }
     }, [data, xScale, yScale]);
 
-    return <path className="path" d={d} />;
+    return (
+        <g tabIndex="0">
+            <desc>{`The linear line representative of the following periods, dates, and obligations: ${description}`}</desc>
```

Comment:
"linear line"?

Reply:
`linear curve` or `linear area`?

In the description I am trying to be descriptive of the type of line or I guess curve or area that is being drawn based off of what I would use if we did not support IE `https://github.com/d3/d3-shape#_area` and I did not have to create the path string myself. The `curve` property of the `d3.area` function would be linear based off of `https://github.com/d3/d3-shape#curves`.

Reply:
Yeah, hmm.... since the line and area both have the same information, maybe the < g class="paths" > should have the tabindex and desc, and not re-group for each < path > ?

Reply:
The data driving the two paths is the same but they are visually different and the reason I gave both paths descriptions relevant to what is visually represented.